### PR TITLE
[feature](json) add json_valid function

### LINKED ISSUE #17216: [Feature] add some new json function
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Description

`json_contains` : [https://dev.mysql.com/doc/refman/5.7/en/json-search-functions.html#function_json-contains](https://dev.mysql.com/doc/refman/5.7/en/json-search-functions.html#function_json-contains)

`json_extract` : support return array, like : [https://dev.mysql.com/doc/refman/5.7/en/json-search-functions.html#function_json-extract](https://dev.mysql.com/doc/refman/5.7/en/json-search-functions.html#function_json-extract)

`json_unqoute` : [https://dev.mysql.com/doc/refman/5.7/en/json-modification-functions.html#function_json-unquote](https://dev.mysql.com/doc/refman/5.7/en/json-modification-functions.html#function_json-unquote)

`json_valid` : [https://dev.mysql.com/doc/refman/5.7/en/json-attribute-functions.html#function_json-valid](https://dev.mysql.com/doc/refman/5.7/en/json-attribute-functions.html#function_json-valid)

`-> operator` : [https://dev.mysql.com/doc/refman/5.7/en/json-search-functions.html#operator_json-column-path](https://dev.mysql.com/doc/refman/5.7/en/json-search-functions.html#operator_json-column-path)

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by nextdreamblue**
# Proposed changes

Issue Number: close #17216

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by nextdreamblue** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.19 seconds
 stream load tsv:          472 seconds loaded 74807831229 Bytes, about 151 MB/s
 stream load json:         39 seconds loaded 2358488459 Bytes, about 57 MB/s
 stream load orc:          67 seconds loaded 1101869774 Bytes, about 15 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230228125322_clickbench_pr_105760.html

**Comment by nextdreamblue** [comment]
run buildall

**Review by xy720 — DISMISSED** [review]
LGTM

File: docs/zh-CN/docs/sql-manual/sql-functions/json-functions/json_valid.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
++------------------+
+| json_valid(NULL) |
++------------------+
+|             NULL |
++------------------+
+1 row in set (0.02 sec)
+```
+
+### keywords
+JSONB, JSON, json_valid
```
Comment (xy720) [inline docs/zh-CN/docs/sql-manual/sql-functions/json-functions/json_valid.md:75]:
```suggestion
JSON, VALID, JSON_VALID
```
Why there is JsonB? seems the argument is always string.

Comment (nextdreamblue) [reply docs/zh-CN/docs/sql-manual/sql-functions/json-functions/json_valid.md:75]: let me fix it

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run p0

**Review by xy720 — APPROVED** [review]

**Pull request merged**